### PR TITLE
Remove RequestContext

### PR DESCRIPTION
**Pull request by jlward**
`RequestContext` is not a thing in Django 1.11.

### CODE REVIEW

File: attachments/templates/attachments/edit_attachment.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% extends "site_base.html" %}
```
Comment (jlward) [inline attachments/templates/attachments/edit_attachment.html]: It should be noted that the changed/additions to templates are only for tests. We don't actually use these templates in practice.

Comment (bocribbz) [reply attachments/templates/attachments/edit_attachment.html]: hmmm this looks confusing from the start.......

Comment (jlward) [reply attachments/templates/attachments/edit_attachment.html]: Well, `edit_attachment.html` didn't exist (it's supposed to). And `site_base.html` doesn't exist (it's supposed to).

File: run_tests.py [hunk 2]
```diff
@@ -26,6 +26,16 @@
         TEST_RUNNER='django_nose.NoseTestSuiteRunner',
         SITE_ID=1,
         MAILER_EMAIL_BACKEND='mailer.tests.TestMailerEmailBackend',
+        TEMPLATES=[
+            {
+                'BACKEND': 'django.template.backends.django.DjangoTemplates',
+                'DIRS': [],
+                'APP_DIRS': True,
```
Comment (bocribbz) [inline run_tests.py]: is this what makes it use the `html` templates in the apps folders vs the haml?

File: attachments/templates/attachments/edit_attachment.html (hunk 1 above)
Comment (bocribbz) [reply attachments/templates/attachments/edit_attachment.html]: why is it suppose to exist? i dont see it being required anywhere

File: run_tests.py (hunk 2 above)
Comment (jlward) [reply run_tests.py]: This is how it knows where to look for templates. It was added in Django 1.7 and required buy the time we get to Django 1.11

File: attachments/templates/attachments/edit_attachment.html (hunk 1 above)
Comment (jlward) [reply attachments/templates/attachments/edit_attachment.html]: Both `create_attachment.html` and `edit_attachment.html` extend from `site_base.html` and both of those are used when GETing their respective views. I added tests around that in this PR. In practice however, PolicyStat passes in a different template to use.

**Comment by bocribbz** [comment]
looks great!

**Pull request merged**